controllers/validation/account_validation_controller.go

```diff
@@ -464,3 +442,19 @@ func (r *ValidateAccount) Reconcile(request reconcile.Request) (reconcile.Result
 	}
 	return utils.DoNotRequeue()
 }
+
+// SetupWithManager sets up the controller with the Manager.
+func (r *AccountValidationReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	r.awsClientBuilder = &awsclient.Builder{}
```

Comment:
```suggestion
	r.awsClientBuilder = &awsclient.Builder{}
	r.OUNameIDMap = map[string]string{}
```

This is required otherwise it will panic from attempting to assign to a nil map

Reply:
Oh, great catch - thanks for spotting it! I must have missed it during the rebase. Added back in now.